R/utils.R

```diff
@@ -5,9 +5,10 @@ parse_text <- function(x) parse(text = x)[[1L]]
 #' @param char A character vector.
 #' @param times an integer giving the number of repetitions.
 #' @return A character vector.
+#' @importFrom purrr map map_chr
 rep_char <- function(char, times) {
-  lapply(times, rep.int, x = char) %>%
-    vapply(paste, collapse = "", character(1L))
+  map(times, rep.int, x = char) %>%
```

Comment:
For the flat serialization, `newlines_and_spaces()` uses `rep_char()` with a vectorised `times` input and for setting spaces at the beginning of comments we do that. I think we can use `map()` in these two places. It speeds things up quite a bit (~15%). Thanks for the hint.